lib/express-metrics.js

```diff
@@ -0,0 +1,31 @@
+var agent = require('./agent');
+var cluster = require('cluster');
+var debug = require('./debug')('express-metrics');
+
+module.exports = function sendExpressMetrics(parentCtl) {
+  if (cluster.isWorker) {
+    agent().on('express:usage-record', forwardRecordToMaster);
+    return;
+  }
+
+  if (!parentCtl) {
+    debug('parentCtl not available, all records will be discarded.');
+    return;
+  }
+
+  cluster.on('fork', function(worker) {
+    worker.on('message', function(msg) {
```

Comment:
This function could be called `forwardRecordToParent`, if you wanted, for parallelism with the other path.